doc/modules/model_persistence.rst

```diff
@@ -89,4 +88,28 @@ another architecture is not supported.
 
 If you want to know more about these issues and explore other possible
 serialization methods, please refer to this
-`talk by Alex Gaynor <https://pyvideo.org/video/2566/pickles-are-for-delis-not-software>`_.
+`talk by Alex Gaynor
+<https://pyvideo.org/video/2566/pickles-are-for-delis-not-software>`_.
+
+Interoperable formats
+---------------------
+
+For production and quality control needs, exporting the model in `Predictive
+Model Markup Language (PMML)
+<http://dmg.org/pmml/v4-4-1/GeneralStructure.html>`_ or `Open Neural Network
+Exchange <https://onnx.ai/>`_ format
+would be a better approach than using `pickle`.
```

Comment:
Phrasing "For production and quality control needs... PMML / ONNX would be a better approach than using pickle": I think that most people who use scikit-learn models in production typically deploy pickled models in docker containers. It's perfectly fine as long as you are aware of the limitations of the pickle format, namely treat the pickled model as a piece of executable software (as is typically the rest of the contents of the docker image) rather than a piece of data or self-decribing source code. For organizations that are used to deploying Python and docker in production this is probably the best approach.

To me the pros of using interoperable formats are:
- make it possible to deploy to platforms that do not have Python installed for any reason (e.g. java-oriented sysadmin culture and JVM-centric production & diagnosis tools),
- decouple the trained model from the specific runtime type and version used to train the model.

E.g. some organizations might decide to allow their datascientist to develop models using any technology (e.g. Python, R, spark) but then only deploy PMML or ONNX models on the inference servers of operational reasons.

Reply:
I've added a line about pickle "containerisation". and reformulate a bit. Your two point are part of the definition of interoperability to me... should I go to this level of detail? Thanks.